pkg/mud/game/game.go

```diff
@@ -15,15 +15,15 @@ type clientCommand struct {
 
 // Game defines the master game object and everything in the game
 type Game struct {
-	Chat        *chat.Chat
+	Chat        stream.Stream
 	CommandChan chan clientCommand
 	everyone    protocol.Sender
 }
 
 // NewGame creates a new game object on the heap
 func NewGame(broadcaster protocol.Sender) *Game {
 	return &Game{
-		Chat:        new(chat.Chat),
+		Chat:        stream.NewStream(100),
```

Comment:
This is a concrete type of stream.Stream, not a stream.Chat or chat.Chat which implements stream.Stream.